### ISSUE TITLE
Position of attachment in body

### ISSUE THREAD

**Issue by muessjoeh** [thunderbird/thunderbird-android#7364]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.711

### Problem you are trying to solve

When you receive an email with an attachment, this attachment is displayed at the very end. When the mail is rather long, for example when there was a long conversation with a lot of answers, this is not unprobable. Because each answer is indented, the mail becomes physically very long on the narrow phone display, having only one letter per line. So you have to scroll down a lot to reach the attachment. 

### Suggested solution

The attachment should at the top, not at the bottom. Or it should be available via a context menu.

**Comment by jmtn** [comment]
This would be an amazing option, I often miss that there is an attachment at all because it appears so far down!

**Comment by DPineault** [comment]
A year has passed and nothing.  Sad.

This is a MAJOR UI mistake and a user experience killer.

Uninstalled.

**Comment by kewisch** [comment]
I agree it is annoying. #592 has some more details on why it isn't easy, I think we can handle it there.
